docker/superset_config.py

```diff
@@ -45,6 +45,9 @@ def get_cache_config():
 # Application Root configuration
 APPLICATION_PREFIX = get_env_variable('APPLICATION_PREFIX')
 
+# WalkMe feature enabled/disbaled
+WALKME_ENABLED = get_env_variable('WALKME_ENABLED')
+
```

Comment:
use boolify before  get_env_variable to convert it in boolean

Reply:
Done